File: CS/CS.NET-VB.NET/ASP.NET 4.0 Book/Source/Chapter 24/CS/BookDetailMVCApp/BookDetailMVCApp/Controllers/BookController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace BookDetailMVCApp.Controllers
{
    public class BookController : Controller
    {
        //
        // GET: /Book/

        public ActionResult Index()
        {
            
            BookDetailMVCApp.Models.ClassBook bs = new BookDetailMVCApp.Models.ClassBook();
            bs.BName = "Black Book ASP.NET";
            bs.BEdition = "2010";
            bs.BPrice = "800";
            bs.BISBN = "111-2222";
            bs.publisher = "Kogent Learning Solutions";

            return View(bs);
        }


    }
}

File: CS/CS.NET-VB.NET/ASP.NET 4.0 Book/Source/Chapter 24/VB/BookDetailMVCAppVB/BookDetailMVCAppVB/Controllers/BookController.vb
Imports System.Web.Mvc

Namespace BookDetailMVCAppVB
    Public Class BookController
        Inherits System.Web.Mvc.Controller

        '
        ' GET: /Book

        Function Index() As ActionResult

            Dim bs As New ClassBook()
            bs.BName = "Black Book ASP.NET"
            bs.BEdition = "2010"
            bs.BPrice = "800"
            bs.BISBN = "111-2222"
            bs.publisher = "Kogent Learning Solutions"

            Return View(bs)

        End Function

    End Class
End Namespace